.golangci.bck.yaml

```diff
@@ -0,0 +1,20 @@
+linters:
```

Comment:
This is the older one, 
We can remove it eventually I guess

Reply:
Let's delete it, it's stored in git

Reply:
Yes, correct